NeuroID/NeuroIDClass/Extensions/NIDSession.swift

```diff
@@ -206,4 +208,37 @@ public extension NeuroID {
 
         return true
     }
+
+    static func startAppFlow(siteID: String, userID: String? = nil) -> SessionStartResult {
+        if NeuroID.clientKey == nil || NeuroID.clientKey == "" {
+            NIDLog.e("Missing Client Key - please call configure prior to calling start")
+            return SessionStartResult(false, "")
+        }
+
+        // immediately flush events before anything else
+        groupAndPOST(forceSend: true)
+
+        // If not started then start
+        var startStatus: SessionStartResult
+        if !NeuroID._isSDKStarted {
+            if userID != nil {
+                startStatus = NeuroID.startSession(userID)
+            } else {
+                let started = NeuroID.start()
```

Comment:
what's the reasoning behind using start() here instead os startSession()? Both would create a new session id right?

Reply:
No, start will not create a sessionID which for some use cases is appropriate (I.e. MGI and Aspiration)